kolibri/plugins/coach/assets/src/views/common/QuizStatus.vue

```diff
@@ -349,7 +349,7 @@
           // palette.red.v_1100 is the darkest available red
           // in the palette. Using this hardcoded color was
```

Comment:
Would you remove this whole comment before we merge? It's not needed anymore, as `$darken` is now supported and documented technique.

Reply:
@MisRob , please check it.

Reply:
Thanks, all good :)!